db/migrate/20230711111829_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text, null: false
+      t.references :user, foreign_key: { on_delete: :cascade }
+      t.references :post, foreign_key: { on_delete: :cascade }
+      t.timestamps
+    end
```

Comment:
* Kindly ensure Table and column names should match the ERD diagram as stated in the [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=Table%20and%20column%20names%20should%20match%20the%20ERD%20diagram.)
![image](https://github.com/ridabensalem/blog-app/assets/52497246/84f7f762-8785-42ed-b855-5a5ae2dd48e5)